release-please-config.json

```diff
@@ -2,6 +2,7 @@
   "bootstrap-sha": "b489df4cf3eae08a25ed401db207a78100cf4fc9",
   "plugins": ["node-workspace"],
   "bump-minor-pre-major": true,
+  "label": "triage:no",
```

Comment:
By the description of [`"label"`](https://github.com/googleapis/release-please/blob/422df5a6a9d44a52c6c682563d1795b8fd246215/schemas/config.json#L431-L434), this might overwrite the default `autorelease: pending` and thus possibly affect the logic. I'm not sure (would need to experiment) if it might be better to rather use [`"extra-label"`](https://github.com/googleapis/release-please/blob/422df5a6a9d44a52c6c682563d1795b8fd246215/schemas/config.json#L73-L76).

Reply:
Weird, `extra-label` isn't documented. I suppose it's safer to try that as a first attempt.